calling the syscall directly for renameat2 breaks fakeroot.
Force fallback til musl has implemented renameat2.

ref: https://gitlab.alpinelinux.org/alpine/aports/-/issues/16016

--- a/lib/renameatu.c
+++ b/lib/renameatu.c
@@ -112,7 +112,7 @@ renameatu (int fd1, char const *src, int
 #if HAVE_WORKING_RENAMEAT2
   ret_val = renameat2 (fd1, src, fd2, dst, flags);
   err = errno;
-#elif defined SYS_renameat2
+#elif defined SYS_renameat2 && defined BREAK_FAKEROOT
   ret_val = syscall (SYS_renameat2, fd1, src, fd2, dst, flags);
   err = errno;
 #endif
